brix11/lib/brix11/projects/gnuace.rb

```diff
@@ -14,17 +14,27 @@ module BRIX11
 
   module Project
 
-    class GnuAce < Handler
+    class GnuMake < Handler
 
-      ID = 'gnuace'
-      DESCRIPTION = 'GNU Make makefiles'
+      ID = 'gnumake'
       BUILDTOOL = 'make'
+      BUILDTOOL_VERSION = `make --version`.match(/[\d.]+/).to_s
```

Comment:
This needs some improvement. In case this runs on a system where GNU make is not installed you will get weird effects when trying to use makefiles to build.

Reply:
Extended with a file existenace check